core/common/hexutil.hpp

```diff
@@ -59,6 +55,13 @@ namespace kagome::common {
    */
   outcome::result<std::vector<uint8_t>> unhex(std::string_view hex);
 
+  /**
+   * @brief Unhex hex-string with 0x in the begining
+   * @param hex hex string with 0x in the beginning
+   * @return unhexed buffer
+   */
+  outcome::result<std::vector<uint8_t>> unhexWith0x(std::string_view hex);
```

Comment:
Remind me please, why have a separate function for this instead of one function that decodes both hex strings with 0x and without?

Reply:
we should be able to do hex and unhex and always get the same result. We might need (although we haven't yet) to hex-encode with prefix, therefore corresponding unhex function is also needed. I don't see many problems keeping that way